scripts/process/process.order_sites.R

```diff
@@ -7,9 +7,22 @@ process.order_sites <- function(viz = as.viz("order-sites")){
   sites <- depends[["nwis-sites"]]
   
   reordered_sites <- sites %>%
-    arrange(desc(dec_long_va))
-    
-  
+    arrange(desc(dec_long_va)) 
   
+  reordered_sites$station_nm <- sapply(reordered_sites$station_nm, siteCapper)
+    
   saveRDS(reordered_sites, viz[['location']])
-}
\ No newline at end of file
+}
+
+siteCapper <- function(x) {
```

Comment:
was this moved from somewhere else? I thought it was in the viz already

Reply:
Could get it by sourcing another file, yeah.